.mvn/extensions.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <extensions>
   <extension>
-    <groupId>org.eclipse.tycho.extras</groupId>
-    <artifactId>tycho-pomless</artifactId>
-    <version>2.5.0</version>
+    <groupId>org.eclipse.tycho</groupId>
+    <artifactId>tycho-build</artifactId>
+    <version>2.7.0</version>
```

Comment:
I think 3.0.0-SNAPSHOT will be needed similarly to Wild Web Developer (for the compare-versions-with-baseline mojo)